fvtest/gctest/main.cpp

```diff
@@ -30,17 +30,14 @@ testMain(int argc, char **argv, char **envp)
 {
 	int result = 1;
 
-	gcTestEnv = (GCTestEnvironment *)testing::AddGlobalTestEnvironment(new GCTestEnvironment(argc, argv));
-	/* This must be called before InitGoogleTest. Because we need to initialize the parameter vector before InitGoogleTest
-	 * calls INSTANTIATE_TEST_CASE_P and takes the vector. */
-	gcTestEnv->GCTestSetUp();
-
 	/*
 	 * ASSERT_NO_FATAL_FAILURE() cannot be used in non-void function. Therefore,
 	 * we need to use HasFatalFailure() to check for fatal failures in sub-routine.
 	 */
 	if (!testing::Test::HasFatalFailure()) {
```

Comment:
Do we still need this check?  I am not sure there is any chance for failure before this point anymore